src/dispatcher.js

```diff
@@ -299,10 +306,19 @@ var dispatcher = {
     return eventCopy;
   },
   getTarget: function(inEvent) {
-
-    // if pointer capture is set, route all events for the specified pointerId
-    // to the capture target
-    return this.captureInfo[inEvent.pointerId] || inEvent._target;
+    var capture = this.captureInfo[inEvent.pointerId];
+    if (!capture) {
+      return inEvent._target;
+    }
+    if (!this.suppressDuringCapture(inEvent)) {
+      return capture;
+    }
+  },
+  suppressDuringCapture: function(inEvent) {
+    var capture = this.captureInfo[inEvent.pointerId];
+    var isNonCaptureTarget = inEvent._target !== capture;
+    var needsSuppressing = BOUNDARY_EVENTS.indexOf(inEvent.type) > -1;
```

Comment:
Since this can be a hot path, I'd like to switch `BOUNDARY_EVENTS` to an object so we don't need to walk through the list for each event.